docs/999-big-list-of-options.md

```diff
@@ -629,7 +629,7 @@ Default: `'auto'`
 
 What export mode to use. Defaults to `auto`, which guesses your intentions based on what the `input` module exports:
 
-* `default` – suitable if you are only exporting one thing using `export default ...`
```

Comment:
There IS an `exportMode: default`. If you want to document `auto`, then you should add a new line, but basically the description of `auto` is given in the paragraph above so I do not think it is needed? In any case, this change is not correct. If `exportMode: default` does not work as intended or is confusing, please let me know.

Reply:
ah, I see. that's indeed somewhat confusing. After your explanation, and re-reading the docs, it makes more sense now. I'm gonna fix this PR with adding a new line, I think that makes it more clear.

One thing what I think is slightly more concerning is the 'auto' mode itself, although in a way I also like it, and I see the intention behind it - although I got bitten by the behavior:

if someone exports a default in 'auto' mode, which will be exported on `module.exports`, and later on also adds one or more named exports, the entire export is changed to be properties of `module.exports` now, and the module would need to be re-published as semver major.

Reply:
This is indeed true, which is why it is strongly recommended not to mix default and named exports and you get a warning if you do in auto mode. If you feel this should be explained better here, suggestions very much welcome.